cluster/ubuntu/config-default.sh

```diff
@@ -29,6 +29,12 @@ export roles_array=($roles)
 
 # Define minion numbers
 export NUM_NODES=${NUM_NODES:-3}
+
+# Define master info if "a/ai" is not appear in roles,
+# node can be dynamically added to or removed from the existed k8s-cluster
+MASTER="vcap@10.10.103.250"
+MASTER_IP="${MASTER#*@}"
+
```

Comment:
Correct me if i'm wrong, seems that these two variables are never referenced. What are they used for?